realtime/server.js

```diff
@@ -1,14 +1,16 @@
-var io = require('socket.io');
-var req = require('request');
-var fs = require('fs');
-var winston = require('winston');
+var io = require('socket.io'); //include websockets
+var req = require('request'); //include request(supports http)
```

Comment:
The purpose of each library can be looked up by name. Furthermore, this library's name is self-descriptive. This comment is unnecessary.